js/script.js

```diff
@@ -563,95 +574,132 @@ function ot_buildMatcher() {
 
-    "ot_linux,ot_on_device,ot_c,ot_X64,ot_CPU":
+    "ot_linux,ot_on_device,ot_c,ot_X64,ot_CPU,ot_stable":
     "Download .tgz file from&nbsp;<a href='https://github.com/microsoft/onnxruntime/releases' target='_blank'>Github</a> <br/>Refer to <a href='http://www.onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements' target='_blank'>docs</a> for requirements.",
 
-    "ot_linux,ot_on_device,ot_cplusplus,ot_X64,ot_CPU":
+    "ot_linux,ot_on_device,ot_cplusplus,ot_X64,ot_CPU,ot_stable":
     "Download .tgz file from&nbsp;<a href='https://github.com/microsoft/onnxruntime/releases' target='_blank'>Github</a> <br/>Refer to <a href='http://www.onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements' target='_blank'>docs</a> for requirements.",
     
-    "ot_linux,ot_on_device,ot_csharp,ot_X64,ot_CPU":
+    "ot_linux,ot_on_device,ot_csharp,ot_X64,ot_CPU,ot_stable":
     "Install Nuget package&nbsp;<a href='https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime.Training' target='_blank'>Microsoft.ML.OnnxRuntime.Training</a>",
 
-    "ot_linux,ot_on_device,ot_c,ot_X64,ot_CUDA":
+    "ot_linux,ot_on_device,ot_c,ot_X64,ot_CUDA,ot_stable":
     "Follow build instructions from&nbsp;<a href='https://onnxruntime.ai/docs/build/training.html' target='_blank'>here</a>",
 
-    "ot_linux,ot_on_device,ot_cplusplus,ot_X64,ot_CUDA":
+    "ot_linux,ot_on_device,ot_cplusplus,ot_X64,ot_CUDA,ot_stable":
     "Follow build instructions from&nbsp;<a href='https://onnxruntime.ai/docs/build/training.html' target='_blank'>here</a>",
     
-    "ot_linux,ot_on_device,ot_csharp,ot_X64,ot_CUDA":
+    "ot_linux,ot_on_device,ot_csharp,ot_X64,ot_CUDA,ot_stable":
     "Follow build instructions from&nbsp;<a href='https://onnxruntime.ai/docs/build/training.html' target='_blank'>here</a>",
 
-    "ot_windows,ot_on_device,ot_c,ot_X64,ot_CUDA":
+    "ot_windows,ot_on_device,ot_c,ot_X64,ot_CUDA,ot_stable":
     "Follow build instructions from&nbsp;<a href='https://onnxruntime.ai/docs/build/training.html' target='_blank'>here</a>",
 
-    "ot_windows,ot_on_device,ot_cplusplus,ot_X64,ot_CUDA":
+    "ot_windows,ot_on_device,ot_cplusplus,ot_X64,ot_CUDA,ot_stable":
     "Follow build instructions from&nbsp;<a href='https://onnxruntime.ai/docs/build/training.html' target='_blank'>here</a>",
     
-    "ot_windows,ot_on_device,ot_csharp,ot_X64,ot_CUDA":
+    "ot_windows,ot_on_device,ot_csharp,ot_X64,ot_CUDA,ot_stable":
     "Follow build instructions from&nbsp;<a href='https://onnxruntime.ai/docs/build/training.html' target='_blank'>here</a>",
 
-    "ot_android,ot_on_device,ot_c,ot_X64,ot_CPU":
+    "ot_android,ot_on_device,ot_c,ot_X64,ot_CPU,ot_stable":
     "Follow installation instructions from&nbsp;<a href='https://onnxruntime.ai/docs/install/#install-for-on-device-training' target='_blank'>here</a>",
 
-    "ot_android,ot_on_device,ot_cplusplus,ot_X64,ot_CPU":
+    "ot_android,ot_on_device,ot_cplusplus,ot_X64,ot_CPU,ot_stable":
```

Comment:
For Android we should point to the android package for all 3 language bindings - c/c++/java.

Reply:
Ah, commenting here for visibility -- leaving the android installation options as is since the C/C++ instructions differ from the Java ones though they all use the same package